lib/galaxy/objectstore/__init__.py

```diff
@@ -1809,6 +1813,9 @@ def ids_per_quota_source(self, include_default_quota_source=False):
             quota_sources[quota_source_label].append(object_id)
         return quota_sources
 
+    def _is_user_defined_source(self, object_store_id: Optional[str]) -> bool:
+        return object_store_id is not None and object_store_id.startswith("user_objects://")
```

Comment:
```suggestion
        return object_store_id and object_store_id.startswith("user_objects://")
```
Because values evaluating to false won't break it

Reply:
I'm afraid mypy doesn't like it :disappointed: 

![image](https://github.com/user-attachments/assets/68e2905e-46e7-450b-92b5-6da78de5f384)